static/dswc_leaderboard.js

```diff
@@ -10,74 +9,93 @@ var dataSource = "";
 
    use file:///..../dev.html in the browser, as it is symlinked to templates/index.html 
    but the below code will switch the API source to Heroku */
-   
+
 if (window.location.href.search(/dev.html/) != -1) {
-    dataSource = "https://dswcregatta.herokuapp.com";
+    dataSource = 'https://dswcregatta.herokuapp.com';
 }
 
 function mkDisplayData(obj, data) {
     var dData = [];
     var fields = [
-        {'key': 'crew', stickyColumn: true},
-        {'key': 'place', class: 'text-center'}, 
-        /* {'key': 'boatNum', class: 'text-center'}, */ 
-        {'key': 'points', class: 'text-center'},
+        { key: 'crew', stickyColumn: true },
+        { key: 'place', class: 'text-center' },
+        /* {'key': 'boatNum', class: 'text-center'}, */
+
+        { key: 'points', class: 'text-center' }
     ];
-    for (var b in data) {
+    for (var result of data) {
         var dRec = {};
-        dRec.crew = data[b].crew.join(', ');
-        /* dRec.boatNum = data[b].boatNum; */
-        dRec.points = data[b].points;
-        dRec.place = data[b].place;
-        dRec._cellVariants = {};
-        for (var r in data[b].races) {
-            var raceStr = parseInt(r, 10) + 1;
-            raceStr = 'R' + raceStr;
+
+        /* dRec.boatNum = result.boatNum; */
+        dRec.place = result.place;
+        dRec.points = result.points;
+        dRec.crew =
+            (result.place === 1
+                ? '🥇'
```

Comment:
The medals aren't rendering on my Chrome browser (on linux), but are rendering on my phone